modules/core/src/main/java/org/apache/ignite/internal/properties/IgniteProductVersion.java

```diff
@@ -109,41 +112,48 @@ public byte maintenance() {
     }
 
     /**
-     * Returns {@code true} if this is a snapshot release, {@code false} otherwise.
+     * Returns the patch version number.
```

Comment:
Please add a description when this method can return `{@code null}`

Reply:
Will it be clearer? `{@code null} if no patch version has been specified.`

Reply:
yes

Reply:
Fix it